src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,24 @@
+package duke.task;
+
+/**
+ * A task of type todo
```

Comment:
Perhaps need a full stop at the end? 🤔
And I noticed the same issue in several other places below.